demo/laneLoader.js

```diff
@@ -264,21 +275,34 @@ function createLaneGeometriesOld(lanes, supplierNum, annotationMode) {
     var geometryRight = new THREE.Geometry();
 
     let left, right, spine;
+    var isContains = false;
```

Comment:
Just a note, but `let` is generally preferred since it narrows the variable's scope (a lot of my older code doesn't follow this and should prob be updated).